### PR TITLE
Automate OCP-27617 and OCP-27686

### PR DESCRIPTION
**Pull request by duanwei33**
pass log:
https://mastern-jenkins-csb-openshift-qe.apps.ocp4.prod.psi.redhat.com/job/Runner-v3-smoke/3170/console
https://mastern-jenkins-csb-openshift-qe.apps.ocp4.prod.psi.redhat.com/job/Runner-v3-smoke/3171/console

@jhou1 @chao007 PTAL

### CODE REVIEW

File: features/storage/csi_clone.feature [hunk 1]
```diff
@@ -183,3 +183,89 @@ Feature: CSI clone testing related feature
...
+  @admin
+  @destructive
+  Scenario: [Cinder CSI Clone] Clone a pvc with default storageclass
+    Given default storage class is patched to non-default
+    And admin clones storage class "my-csi-default" from "standard-csi" with:
+      | ["metadata"]["name"] | my-csi-default |
+    When I run the :patch admin command with:
+      | resource      | storageclass                                                                           |
+      | resource_name | my-csi-default                                                                         |
+      | p             | {"metadata": {"annotations": {"storageclass.kubernetes.io/is-default-class": "true"}}} |
```
Comment (liangxia) [inline features/storage/csi_clone.feature:197]: Above two steps can be simplified with
```
    And admin clones storage class "xxx" from "yyy" with:
      | ["metadata"]["annotations"]["storageclass.kubernetes.io/is-default-class"] | "true" |
```

Comment (duanwei33) [reply features/storage/csi_clone.feature:197]: @liangxia Thanks a lot for the suggestion, actually I tried before this PR and got failure as below:

>     And admin clones storage class "my-csi-default" from "standard-csi" with: # features/step_definitions/storage_class.rb:174
>       #  | ["metadata"]["name"] | my-csi-default |
>       #When I run the :patch admin command with:
>       #  | resource      | storageclass                                                                           |
>       #  | resource_name | my-csi-default                                                                         |
>       #  | p             | {"metadata": {"annotations": {"storageclass.kubernetes.io/is-default-class": "true"}}} |
>       | ["metadata"]["annotations"]["storageclass.kubernetes.io/is-default-class"] | "true" |
>       [10:15:55] INFO> Shell Commands: oc get StorageClass standard-csi -o yaml --kubeconfig=/home/wduan/workdir/MINT-wduan/ocp4_admin.kubeconfig
>       allowVolumeExpansion: true
>       apiVersion: storage.k8s.io/v1
>       kind: StorageClass
>       metadata:
>         creationTimestamp: "2021-06-14T03:52:12Z"
>         managedFields:
>         - apiVersion: storage.k8s.io/v1
>           fieldsType: FieldsV1
>           fieldsV1:
>             f:allowVolumeExpansion: {}
>             f:provisioner: {}
>             f:reclaimPolicy: {}
>             f:volumeBindingMode: {}
>           manager: openstack-cinder-csi-driver-operator
>           operation: Update
>           time: "2021-06-14T03:52:12Z"
>         name: standard-csi
>         resourceVersion: "556281"
>         uid: 1f396987-d581-438e-ae20-8c25c53e69ff
>       provisioner: cinder.csi.openstack.org
>       reclaimPolicy: Delete
>       volumeBindingMode: WaitForFirstConsumer
>       
>       [10:15:57] INFO> Exit Status: 0
>       undefined method `[]=' for nil:NilClass (NoMethodError)
>       (eval):1:in `block (2 levels) in <top (required)>'
>       /home/wduan/01Work/github/cucushift/verification-tests/features/step_definitions/storage_class.rb:202:in `eval'
>       /home/wduan/01Work/github/cucushift/verification-tests/features/step_definitions/storage_class.rb:202:in `block (2 levels) in <top (required)>'
>       /home/wduan/01Work/github/cucushift/verification-tests/features/step_definitions/storage_class.rb:201:in `each'
>       /home/wduan/01Work/github/cucushift/verification-tests/features/step_definitions/storage_class.rb:201:in `/^admin clones storage class "(.+?)" from "(.+?)" with:$/'
>       features/storage/csi_clone.feature:192:in `admin clones storage class "my-csi-default" from "standard-csi" with:'

When looking into the step(https://github.com/openshift/verification-tests/blob/master/features/step_definitions/storage_class.rb#L174-L223), looks like the source storage class should have the annotations.storageclass.kubernetes.io/is-default-class defined, but in my case the standard-csi doesn't have and it doesn't work. Could you help confirm?

Comment (liangxia) [reply features/storage/csi_clone.feature:197]: Recheck the scenario.   First you made the default storage class to non-default, then clone a storage class (no diff except the name) from the previous storage class, and made the newly created one as default. 
This looks strange to me. I think those 3 steps are not necessary.

Comment (duanwei33) [reply features/storage/csi_clone.feature:197]: As in-tree plugin doesn't support clone, this test is to use csi-driver as default sc when creating the clone pvc.

Comment (liangxia) [reply features/storage/csi_clone.feature:197]: Got it.

File: features/storage/csi_clone.feature [hunk 2]
```diff
@@ -183,3 +183,89 @@ Feature: CSI clone testing related feature
...
+      | sh | -c | echo "clone test" > /mnt/local/testfile |
+    Then the step should succeed
+    And I execute on the pod:
+      | sh | -c | sync -f /mnt/local/testfile |
+    Then the step should succeed
+
+    # Clone mypvc-ori without sc specified
+    Given I obtain test data file "storage/csi/pvc-clone.yaml"
+    When I create a dynamic pvc from "pvc-clone.yaml" replacing paths:
+      | ["metadata"]["name"]                         | mypvc-clone  |
```
Comment (liangxia) [inline features/storage/csi_clone.feature:222]: Please fix the extra spaces issue in the tables.
LGTM other than that.

Comment (duanwei33) [reply features/storage/csi_clone.feature]: Fixed.

**Comment by duanwei33** [comment]
@liangxia Thanks for the comments, fixed the format isue, PTAL.

**Pull request merged**